sql/src/main/java/io/crate/execution/engine/collect/collectors/OrderedLuceneBatchIteratorFactory.java

```diff
@@ -79,7 +84,10 @@
                 collectorsByShardId = null;
```

Comment:
shouldn't the pagingIterator (`requiresScroll=true`) here also be wrapped into a RamAccountingPagingIterator?